### DISCUSSION TITLE
Security issue?

### DISCUSSION THREAD

**Discussion by fgreco55** [community/community#54270] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've been getting a few " 12345 is your GitHub authentication code" texts recently all without me attempting to login.  They are coming from a number ending in x9562.  Is anyone else getting these?

**Comment by airtower-luna** [comment]
Did you set up SMS for 2FA? If yes, my guess is that someone managed to steal or guess your password and is trying to log in with it. Obviously I can't say for sure if that's actually the case, but it'd fit. *Assuming* that's what is going on, changing your password would make sense.

**Reply by fgreco55** [reply]
Yes, I have SMS/2FA setup.  I'm definitely changing my passwd.   Appreciate the response.
